tools/wpt/browser.py

```diff
@@ -644,10 +643,21 @@ def find_webdriver(self, channel=None):
     def install_webdriver(self, dest=None, channel=None):
         raise NotImplementedError
 
-    def version(self, binary):
+    def version(self, binary=None, webdriver_binary=None):
+        if webdriver_binary is not None:
+            try:
+                output = call(webdriver_binary, "--version").strip()
+                # TODO: explain
+                if output.startswith('Included with '):
+                    output = output[14:]
+                return output
+            except subprocess.CalledProcessError:
+                # TODO stderr gets printed :'(
```

Comment:
I've found that `call` doesn't suppress stderr and just prints it instead. This isn't great because this is expected to fail on every `safaridriver` before STP 67, notable including Safari 12 stable. Should I just change `call` to return a stdout,stderr tuple?

Reply:
That's surprising; it looks like it calls `subprocess.checkou_output` with no arguments. I would suggest making it take `**kwargs` and passing those down to the caller so you can redirect stderr to `/dev/null` or whatever.